docs/fastdds/xtypes/idl_parsing.rst

```diff
@@ -0,0 +1,63 @@
+.. include:: ../../03-exports/aliases.include
+.. include:: ../../03-exports/aliases-api.include
+.. include:: ../../03-exports/roles.include
+
+.. _dynamic-types-idl-parsing:
+
+Dynamic Types IDL Parsing
+=========================
+
+*Fast DDS* supports the implementation of |DynamicTypes| for parsing IDL files at runtime to generate
+dynamic data types. The |DynamicTypeBuilderFactory::create_type_w_uri| API allows users to provide a URI
```

Comment:
In our guidline of coding style, for markdown files, each sentence must be in a separate line.